test/app/app.ts

```diff
@@ -16,11 +16,13 @@ datadogLogs.init({
 })
 
 datadogRum.init({
+  allowedTracingOrigins: [new RegExp(hostname)],
```

Comment:
`new RegExp(hostname)` is a bit scary as its behavior is unpredictable (it changes according to the domain). Maybe be more explicit?
```suggestion
  allowedTracingOrigins: typeof location === 'object' ? [location.origin] : undefined,
```

Reply:
not sure to get your point, `location.origin` also changes according to the domain.
anyway, it is the npm test app configuration, so is it relevant?

Reply:
`new RegExp(hostname)` raises a lot of questions:

* what if hostname is not a valid regexp? Is it possible? should I care?
* what is the impact of dots in the hostname?
* because the regexp has no `^` and `$`, it matches any domain that contains the hostname, is it expected?

It will probably not have any impact on our tests, but nonetheless I don't think it should be shown as an acceptable solution anywhere in our code (even if this is not directly documentation it should reflect good practices).

Reply:
I don't think it worth to discuss it further, I'll do the change